Entities/Items/Crate/Crate.as

```diff
@@ -294,11 +294,15 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 	}
 	else if (isUnpacking(this))
 	{
-		caller.CreateGenericButton("$DISABLED$", buttonpos, this, this.getCommandID("stop unpack"), getTranslatedString("Stop {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name"))));
+		CButton@ button = caller.CreateGenericButton("$DISABLED$", buttonpos, this, this.getCommandID("stop unpack"), getTranslatedString("Stop {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name"))));
+		button.enableRadius = 20.0f;
 	}
 	else if (hasSomethingPacked(this))
 	{
-		caller.CreateGenericButton(12, buttonpos, this, this.getCommandID("unpack"), getTranslatedString("Unpack {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name"))));
+		CBitStream params;
+		params.write_u16( caller.getNetworkID() );
```

Comment:
```suggestion
		params.write_u16(caller.getNetworkID());
```

Reply:
I have seen white-space being done in `if ( )` statements for readability.
Should it not be done at all, or in if statements only?